rusttorney-server/src/server.rs

```diff
@@ -114,35 +222,50 @@ impl<'a> AOServer<'a> {
     pub async fn run(&mut self) -> anyhow::Result<()> {
         use futures::StreamExt;
 
+        log::debug!("Beginning migration...");
         self.migrate().await?;
```

Comment:
Migration is a something that can (potentially) break something, so shouldn't we warn the user & process only with his blessing?